### PR TITLE
Replace sphynx syntax with markdown.

### PR DESCRIPTION
**Pull request by schmmd**
`find . -name '*.py' -exec gsed -i 's/:[a-z][a-z]*:~*//g' \{\} \;`

### CODE REVIEW

File: allennlp/commands/__init__.py [hunk 1]
```diff
@@ -73,10 +73,10 @@ def create_parser(prog: Optional[str] = None) -> argparse.ArgumentParser:
 
 def main(prog: Optional[str] = None) -> None:
     """
-    The :mod:`~allennlp.run` command only knows about the registered classes in the ``allennlp``
+    The `allennlp.run` command only knows about the registered classes in the ``allennlp``
```
Comment (matt-gardner) [inline allennlp/commands/__init__.py]:
```suggestion
    `allennlp.run` only knows about the registered classes in the ``allennlp``
```

File: allennlp/commands/find_learning_rate.py [hunk 2]
```diff
@@ -261,11 +261,11 @@ def search_learning_rate(
     stopping_factor: float = None,
 ) -> Tuple[List[float], List[float]]:
     """
-    Runs training loop on the model using :class:`~allennlp.training.trainer.GradientDescentTrainer`
+    Runs training loop on the model using `allennlp.training.trainer.GradientDescentTrainer`
     increasing learning rate from ``start_lr`` to ``end_lr`` recording the losses.
     # Parameters
```
Comment (matt-gardner) [inline allennlp/commands/find_learning_rate.py]:
```suggestion
    Runs training loop on the model using
    [`GradientDescentTrainer`](../../training/trainer/#gradientdescenttrainer) increasing learning
    rate from `start_lr` to `end_lr` recording the losses.
    
    # Parameters
```

**Comment by schmmd** [comment]
@matt-gardner if you're interested in merging this let me know and I'll go through it in detail.  I didn't apriori because I knew we might abandon it.

**Comment by matt-gardner** [comment]
I think the kinds of changes I proposed are the right solution here, and someone should do it before our 1.0 release.  I'm happy to do some, just not today.

**Comment by schmmd** [comment]
Ah, I understand how you'd like to move forward now. Thanks.

**Pull request closed without merging**